templates/default/custom.properties.erb

```diff
@@ -301,10 +304,13 @@ org.opencastproject.staticfiles.webserver.enabled=false
 # for the uploaded static file. If not set Opencast uses endpoints to serve and secure the files.
 #org.opencastproject.staticfiles.webserver.url=${org.opencastproject.server.url}/staticfiles/
 
-# Timeout for capture agent status, in minutes.
-# Capture agents which have not sent status updates for this period will be marked as offline.
-# Default: 120 minutes (2 hours)
-#org.opencastproject.capture.admin.timeout=120
+# This is the maximum allowable size in bytes for a file to be uploaded. If the property is missing or set to 0 it is
+# disabled.
+# Default is 1000000000 which is 1GB.
```

Comment:
This seems to contradict the previous comment. If the property is missing is it disabled or is it set to the default of 1GB?

Reply:
@lbjay you're right. That's a typo in OC text. The default is 1GB if the param is missing. Only if set to 0 is the max size disabled.


  /** The maximum file size to allow to be uploaded in bytes, default 1GB */
private long maxUploadSize = 1000000000;

https://github.com/opencast/opencast/blob/develop/modules/static-file-service-api/src/main/java/org/opencastproject/staticfiles/endpoint/StaticFileRestService.java#L117

    if (cfgMaxUploadSize.isSome())
         maxUploadSize = Long.parseLong(cfgMaxUploadSize.get());

https://github.com/opencast/opencast/blob/develop/modules/static-file-service-api/src/main/java/org/opencastproject/staticfiles/endpoint/StaticFileRestService.java#L149